docs/developers/tensorflow_coverage/tflite_base_converage.md

```diff
@@ -0,0 +1,165 @@
+# TensorFlow Lite Base Operation Coverage
+
+Tests of the TFLite's [builtin](https://www.tensorflow.org/mlir/tfl_ops)
+operation coverage.
+
+This focuses on lowering to `tosa` operations and may not have support for
+all IREE backends.
+
+*Last Updated: 2022/5/25
+
+## TFlite Operation Coverage
+
+| TFLite Operation | Static | Dynamic |
+|:----------:|:--------------------------------------------:|:--------------------------------------------:|
```

Comment:
Did you intentionally center align the left column? Left align might look better IMO

https://www.pluralsight.com/guides/working-tables-github-markdown

Reply:
Nope, just copied and tweaked. Updated for left alignment.